### DISCUSSION TITLE
Ability to have multiple trigger phrases; each with it's own formatting settings

### DISCUSSION THREAD

**Discussion by tcallahaneis** [argenos/nldates-obsidian#136] [Ideas]
I currently have NLD set up to output YYYY-MM-DD when I use the trigger phrase '@'. There are times I would like to include the time as well. I think a nice enhancement would be to allow multiple trigger phrases, each with it's own format. That way I could set up another trigger phrase, e.g., '@@' to insert date and time formatted like YYYY-MM-DD HH:mm.

**Comment by nhan000** [comment]
I need this too. Or is there a way to install a duplicated version of this plugin? I followed the instructions here but it didn't work https://forum.obsidian.md/t/how-to-install-a-duplicate-plugin-i-e-the-same-plugin-twice-under-a-different-name/42863/2

**Comment by jasonhaslow-ui** [comment]
This would be great.  My use case is one trigger to include wikilink and a different trigger to not include wikilink.   I know there's a preference to set this, but I use both formats often so am either adding the link or removing the link manually quite often.
